### PR TITLE
style fixes in PLUMED easyconfig

### PR DESCRIPTION
**Pull request by boegel**
follow-up for #1596, fix remarks by @fgeorgatos

### CODE REVIEW

File: easybuild/easyconfigs/p/PLUMED/PLUMED-2.0.4-ictce-7.1.2.eb [hunk 1]
```diff
@@ -1,25 +1,22 @@
-# This is a quickly made easyconfig, using some dirty tricks... 
-# Warning: Here be dragons
 # by Ward Poelmans <wpoely86@gmail.com>
 
 easyblock = 'ConfigureMake'
 
-name = 'plumed'
+name = 'PLUMED'
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/p/PLUMED/PLUMED-2.0.4-ictce-7.1.2.eb]: VERY good catch; thanks, it looks good now!

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/e8f0bd4e93bed226e9bf for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/3b6b8e978fb2d0ddb5cf for a full test report.

**Comment by boegel** [comment]
Thanks for the review @fgeorgatos!

**Pull request merged**